ocp_resources/virtual_machine.py

```diff
@@ -162,3 +162,14 @@ def printable_status(self):
             VM printableStatus if VM.status.printableStatus else None
         """
         return self.instance.get("status", {}).get("printableStatus")
+
+    def wait_for_status_null(self, status, timeout=TIMEOUT_4MINUTES):
+        LOGGER.info(f"Wait for {self.kind} {self.name} status {status} to be None")
```

Comment:
`to be None` it should be None or null?
function name have null and the logger None

Reply:
right, should be None (not exist/empty...)